tree/dataframe/inc/ROOT/RActionBase.hxx

```diff
@@ -79,4 +79,3 @@ public:
 } // ns ROOT
 
 #endif // ROOT_RACTIONBASE
-
```

Comment:
If you are splitting the header files in any case, why not add them already formatted by clang when splitting?